tests/LineBot.Tests/RichMenuRequestTests/TheSizePropertyTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright 2017-2018 Dirk Lemstra (https://github.com/dlemstra/line-bot-sdk-dotnet)
```

Comment:
This file should be renamed to `tests/LineBot.Tests/RichMenu/RichMenuTests/TheSizeProperty.cs`.